golang/pkg/clusterpb/cluster.pb.go

```diff
@@ -19,7 +19,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
 // 	protoc-gen-go v1.28.1
-// 	protoc        v3.20.1
+// 	protoc        v4.25.1
```

Comment:
The version of `protoc` is `v4.25.1` rather than `v3.25.1`, is this as expected?

Reply:
As long as Golang compiled, it should be fine.

I'm going to add some checks on this.

Reply:
As you can see, the generated code is not changed

Reply:
No problem, but the version of `protoc` in `README.md` is `v3.25.1`,the generate code is `v4.25.1`, it is confusing, should we add some comment to explain this question?

Reply:
Yes. We should use https://github.com/protocolbuffers/protobuf/releases/tag/v25.1. Let me update it.

Reply:
Updated.